src/IntervalSets.jl

```diff
@@ -229,6 +229,13 @@ Clamp the scalar `t` such that the result is in the interval `i`.
 clamp(t, i::TypedEndpointsInterval{:closed,:closed}) =
     clamp(t, leftendpoint(i), rightendpoint(i))
 
+"""
+    mod(x, i::ClosedInterval)
+
+Find `y` in the `i` interval such that `x ≡ y (mod n)`, where `n = width(i)`.
+"""
```

Comment:
````suggestion
Find `y` in the `i` interval such that `x ≡ y (mod n)`, where `n = width(i)`.

# Examples

```jldoctest
julia> I = 2.5..4.5;

julia> mod(3.0, I)
3.0

julia> mod(5.0, I)
3.0

julia> mod(2.5, I)
2.5

julia> mod(4.5, I)  # (a in I) does not imply (a == mod(a, I))
2.5
```
"""
````